docs/development-guide.md

```diff
@@ -113,11 +140,33 @@ This doc explains how to build and run the Online Boutique source code locally u
    This will build and deploy the application. If you need to rebuild the images
    automatically as you refactor the code, run `skaffold dev` command.
 
+   **If you encounter build errors** (see Troubleshooting section below), use pre-built images instead:
+   ```shell
+   kubectl apply -f ./release/kubernetes-manifests.yaml
+   ```
```

Comment:
I would remove this text. This doc is for developing on Bank of Anthos, so pulling pre-built images defeats the purpose of this guide)